### PR TITLE
fix(cli): v1 migration should add the updater plugin if it is enabled

### PR DESCRIPTION
**Pull request by lucasfernog**

### CODE REVIEW

File: tooling/cli/src/migrate/migrations/v1/config.rs [hunk 1]
```diff
@@ -114,6 +114,10 @@ fn migrate_config(config: &mut Value) -> Result<MigratedConfig> {
       process_updater(tauri_config, &mut plugins)?;
     }
 
+    if plugins.contains_key("updater") {
```
Comment (FabianLars) [inline tooling/cli/src/migrate/migrations/v1/config.rs]: shouldn't this be `if let Some(updater) = tauri_config.remove("updater") {` (moved into the above block)?

Comment (lucasfernog) [reply tooling/cli/src/migrate/migrations/v1/config.rs]: I didn't want to add the migrated argument (lazy) but it's indeed cleaner that way.. pushed

Comment (FabianLars) [reply tooling/cli/src/migrate/migrations/v1/config.rs]: ah tbh i just missed the process_updater line above that (its comment is saying `cli` btw) so i thought the code is actually incorrect.

File: tooling/cli/src/migrate/migrations/v1/config.rs [hunk 2]
```diff
@@ -111,7 +111,7 @@ fn migrate_config(config: &mut Value) -> Result<MigratedConfig> {
       }
 
       // cli
```
Comment (FabianLars) [inline tooling/cli/src/migrate/migrations/v1/config.rs]:
```suggestion
      // updater
```

**Review by FabianLars — APPROVED** [review]

**Pull request merged**